src/sdk/pynni/nni/compression/torch/utils.py

```diff
@@ -0,0 +1,280 @@
+import os
+import sys
+import yaml
+import logging
+import random
+import numpy as np
+
+from os import path as osp
+from collections import OrderedDict
+
+import pandas as pd
+import torch
+import datetime
+
+class AverageMeter:
```

Comment:
We should not put these utility classes/functions not related to model compression into model compression SDK